main/fight.py

```diff
@@ -1,19 +1,40 @@
 from character import Character
+from random import randint
 
 
 class Fight:
```

Comment:
Look at those methods, each of them needs `player` and `target`  as an input. Why not to use `__init__`  with player and target and then reference to them in methods? It will be more clean
```suggestion
class Fight:
def __init__(self, player: Character, target: Character):
    self.player = player
    self.target = target

```